pajbot/modules/ascii.py

```diff
@@ -105,16 +113,18 @@ def on_pubmsg(self, source, message, **rest):
         if AsciiProtectionModule.check_message(message) is False:
             return
 
-        duration, punishment = self.bot.timeout_warn(
-            source, self.settings["timeout_length"], reason=self.settings["timeout_reason"]
-        )
+        if self.settings["moderation_action"] == "Delete":
+            self.bot.delete_message(tags["id"])
```

Comment:
We should return False here too to stop any further parsing of modules (could just use the same return False as below